### PR TITLE
Fix text with code tag becoming invisible in dark mode

### PR DESCRIPTION
**Pull request by humble-barnacle001**
See #4226

### CODE REVIEW

**Comment by Trott** [comment]
Welcome, @humble-barnacle001, and thanks for the pull request. This doesn't seem to correct the contrast issue for me. Is it possible that not all the changes were checked in or something like that?

Without this PR:

image

With this PR:

image

**Comment by humble-barnacle001** [comment]
I think the issue is fixed in 3640e3f. I forgot to checkout the latest changes to the upstream. Thanks for the correction suggestion @Trott

**Review by Trott — APPROVED** [review]
This fixes most instances. There is still a low-contrast issue in two cases on the page. It would be nice to fix those, but given that this fixes 30+ other instances, I'm ok with landing it and saving the subsequent fix for later PRs.

**Comment by Trott** [comment]
@DerekNonGeneric Any chance you have some time to review this?

File: layouts/css/layout/_dark-theme.scss [hunk 1]
```diff
@@ -8,6 +8,12 @@
     color: $white;
   }
 
+  *{
```
Comment (Trott) [inline layouts/css/layout/_dark-theme.scss]:
```suggestion
* {
```

**Comment by humble-barnacle001** [comment]
@Trott could you please put up a link to the page and/or screenshot so that I can look into it?

**Comment by humble-barnacle001** [comment]
Could not apply b52b573 due to lint issue

**Review by DerekNonGeneric — APPROVED** [review]
RSLGTM

**Comment by Trott** [comment]
> @Trott could you please put up a link to the page and/or screenshot so that I can look into it?

The word "Example" here is readable but doesn't quite meet WCAG AA standards for accessible contrast.

image

Obviously, even without fixing that issue, this change is a vast improvement, so I'm OK with it landing without fixing that much less severe issue.

**Comment by Trott** [comment]
> The word "Example" here is readable but doesn't quite meet WCAG AA standards for accessible contrast.

WCAG requires 4.5 contrast ratio and that text has a contrast ratio of 4.45, so it's a borderline case!

**Comment by humble-barnacle001** [comment]
I shall look into it, but should I put it as a seperate PR or in the same one?

**Comment by Trott** [comment]
> I shall look into it, but should I put it as a seperate PR or in the same one?

Let's make it a separate PR. I'll land this one now.

**Pull request merged**